### ISSUE TITLE
Race condition -> redis.exceptions.BusyLoadingError

### ISSUE THREAD

**Issue by fxkr** [yahoo/redislite#98]
If I have a DB with some items in it, sometimes it takes redis too long to load them, and redislite fails with:

```
Traceback (most recent call last):
  File "temp.py", line 17, in <module>
    db = redislite.StrictRedis('testdb.redis')
  File "/home/fxkr/.local/lib/python2.7/site-packages/redislite/client.py", line 338, in __init__
    self.ping()
  File "/usr/lib/python2.7/site-packages/redis/client.py", line 674, in ping
    return self.execute_command('PING')
  File "/usr/lib/python2.7/site-packages/redis/client.py", line 571, in execute_command
    return self.parse_response(connection, command_name, **options)
  File "/usr/lib/python2.7/site-packages/redis/client.py", line 577, in parse_response
    response = connection.read_response()
  File "/usr/lib/python2.7/site-packages/redis/connection.py", line 569, in read_response
    response = self._parser.read_response()
  File "/usr/lib/python2.7/site-packages/redis/connection.py", line 241, in read_response
    raise error
redis.exceptions.BusyLoadingError: Redis is loading the dataset in memory
```

To reproduce, generate a DB:

```py
import redislite, os
db = redislite.StrictRedis('bug.redis')
for key in xrange(10000):
    db.hset("h1", os.urandom(32), " " * 65536)
db.save()
```

Then try to load it again. This will fail in most cases (on my machine):

```py
db = redislite.StrictRedis('bug.redis')
```

**Comment by dwighthubbard** [comment]
Good catch, the code to reproduce the issue is really helpful.

**Comment by dwighthubbard** [comment]
This issue should be fixed in Release 3.0.287
Please let me know if there are still issues.
